src/components/network/layers/fork-line-layer.js

```diff
@@ -32,20 +34,32 @@ export default class ForkLineLayer extends LineLayer {
             'vs:#decl': `
 attribute float instanceLineParallelIndex;
 attribute float instanceLineAngle;
+attribute float instanceOffsetStart;
 uniform float distanceBetweenLines;
 uniform float maxParallelOffset;
 uniform float minParallelOffset;
+uniform float substationRadius;
+uniform float substationMaxPixel;
             `,
             'float segmentIndex = positions.x': `;
 target = source ;
 if( abs(instanceLineParallelIndex) != 9999. ) {
     float offsetPixels = clamp(project_size_to_pixel( distanceBetweenLines), minParallelOffset, maxParallelOffset );
     float offsetCommonSpace = project_pixel_size(offsetPixels);
-    vec4 trans = vec4(cos(instanceLineAngle), -sin(instanceLineAngle ), 0, 0.) * instanceLineParallelIndex ;
+
+    float offsetSubstation = clamp(project_size_to_pixel(substationRadius*instanceOffsetStart ), 
+                                    1., 
+                                    substationMaxPixel * instanceOffsetStart );
+    float offsetSubstationCS = project_pixel_size(offsetSubstation) ;
```

Comment:
It took me time to understand that CS means CommonSpace, I think it's best to use the explicit form ? At lease the variables OffsetCommonSpace and OffsetSubstationsCS should use the same naming convention